routes/user.route.js

```diff
@@ -8,13 +8,13 @@ const router = Router();
 router.post('/login', noAuth, UserController.login);
 router.post('/register', noAuth, UserController.create);
 router.get('/logout', stdAuth, UserController.logout);
-router.post('/registerAdmin', adminAuth, UserController.createAdmin);
+router.post('/registerAdmin',  UserController.createAdmin);
```

Comment:
Shouldn't be able to create an admin via the API, it should be done in the db, and this would allow anyone to create an admin